docs/superpowers/specs/2026-04-30-autoplay-design.md

```
@@ -0,0 +1,258 @@
+- Set in `PerformanceStage` after constructing `_judgementManager`: `_judgementManager.IgnorePlayerInput = _autoPlayEnabled;`
+
+**Disposal** — in `CleanupGameplayManagers`:
+```csharp
+_chipSoundCache?.Dispose();
+_chipSoundCache = null;
+```
+
+## Data Flow
+
+**Chart load:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)